tests/test_dns.py

```diff
@@ -117,7 +117,7 @@ async def test_resolve_hostaddr_async_bad(
         for k, v in env.items():
             monkeypatch.setenv(k, v)
     params = conninfo_to_dict(conninfo)
-    with pytest.raises((TypeError, psycopg.Error)):
+    with pytest.raises(psycopg.Error):
```

Comment:
Maybe I had some test in the past raising a legit TypeError but they didn't make to the commit. I can't remember.